index/secondary-index/src/test/scala/org/apache/carbondata/spark/testsuite/secondaryindex/TestSIWithSecondryIndex.scala

```diff
@@ -61,6 +62,16 @@ class TestSIWithSecondryIndex extends QueryTest with BeforeAndAfterAll {
     }
   }
 
+  test ("test alter drop columns on the SI columns") {
+    sql("create table test1 (name string, id string, country string) stored as carbondata")
```

Comment:
please change table name, and give some meaningful name with issue

Reply:
modified